app/commands/notification/create.rb

```diff
@@ -6,10 +6,13 @@ class Create
 
     def call
       klass = "notifications/#{type}_notification".camelize.constantize
-      klass.create!(
-        user: user,
-        params: params
-      )
+
+      klass.create!(user: user, params: params).tap do
+        NotificationsChannel.broadcast_to(user, {
+                                            type: "notifications.changed",
+                                            payload: { count: user.notifications.unread.count }
+                                          })
```

Comment:
I think this can be refactored slightly later: https://github.com/exercism/v3-website/issues/84

Reply:
I'll add this in now.